debian/control

```diff
@@ -38,7 +38,8 @@ Depends:
     kano-toolset,
     kano-touch-support,
     kano-os-homedir-content,
-    kano-notifications (>= 4.2.1)
+    kano-notifications (>= 4.2.1),
+    python3-docopt
```

Comment:
We should add a dependency on `mercury` and the python wrappers for it.

Reply:
Added